### ISSUE TITLE
Grant Task 2a: Parameter fitting in the Web Lab

### ISSUE THREAD

**Issue by jonc125** [ModellingWebLab/project_issues#4]
The tools that we are already building for fitting distributions of model parameters from experimental data (described above) will be incorporated into the Web Lab, and a user interface for fitting models to uploaded data sets will be developed. A language we are creating for describing the parameter fitting process will be finalised and used to record parameter fitting for the case studies.

**Comment by MichaelClerx** [comment]
See: https://github.com/martinjrobins/pints and Aidan's dphil work

**Comment by mirams** [comment]
Yep - we should think a bit about how `pints' relates to the Web Lab fitting description. I'm envisaging it being somewhere that we easily try out different approaches, but can perhaps easily export a Web Lab compatible fitting description (We need a name for this by the way - Models, Protocols, ???. It wants to encapsulate model validation and selection aspects, so probably not just 'Fits', unless those are separate concepts...).
